planet/src/main/webapp/places.js

```diff
@@ -315,9 +342,32 @@ function savePlace(x) {
     if(x.innerHTML === "favorite_border") {
         // Set as saved
         x.innerHTML = "favorite";
+        // Save place to database
+        let name = document.getElementById('place-name').innerHTML; 
+        let address = document.getElementById('place-address').innerHTML; 
+        let open = document.getElementById('openingTime').innerHTML;
+        let close = document.getElementById('closingTime').innerHTML;
+        updateDatabase('test', name, address, open, close); 
     }
     else {
         // Set as unsaved
         x.innerHTML = "favorite_border";
+        // Delete place from database
+        deletePlace('test');
     }
 }
+
+/** Update database and add place with information */
+function updateDatabase(placeID, name, address, open, close) {
+    database.ref('places/' + placeID).set({
+        name: name,
+        address: address,
+        openingTime: open,
+        closingTime: close
+    });
+}
+
```

Comment:
Do we want to write to the update the users tree as well?

Reply:
Yes! I will do that as a next step, still trying to figure out how to access place ID!